<launch>
    <param	name="uart_num"	type="string" value="/dev/ttyUSB0"	/>
    <node
        pkg="dtu"
        type="dtu_node"
        name="dtu_node"
        output="screen"
        cwd="node">
    </node>
</launch>
